src/source_list.rs

```diff
@@ -0,0 +1,44 @@
+//! Endpoint -> Famedly Zitadel sync tool.
```

Comment:
Maybe `DeactivationList`?

Reply:
I would probably also go with `source/endpoint.rs`. It looks self-explanatory:
- `source/ldap.rs`
- `source/endpoint.rs`
- `source/google.rs`
- etc.

The only question here is, how abstract we really can get. Sure, it's nice to have an `endpoint` source super generic, but the truth is, that every client who provides such an endpoint, can come up with different implementation. UKT gives a list of emails for deletion. Some other client can give user_ids (which is the case of ldap as well -> which causes a problem for a generic trait that expects to have a list of deletions defined in one way).

Reply:
In the end, I went with `source/ukt.rs`. As pointed out elsewhere, the implementation needs to be quite specific for the provider, thus no sense in pushing it for super-abstractions IMO.

Reply:
`ukt.rs` isn't particularly descriptive of what the source actually is, though. `endpoint.rs` meanwhile just means a generic API endpoint, which is far too generic of a name, I mean, technically all other sources will be endpoints too.

I know we're quibbling about names here, and I agree `ukt.rs` is at least better than a name which implies you can do anything with it, but something more descriptive like `http-deactivation-list.rs` or `http-email-list.rs` just seems clearer, and avoids mentioning specific clients in public?

Reply:
Done.

Reply:
The filename is `sources/ukt.rs`, the struct name is `UktSource` and has a field `ukt_config: UktSourceConfig`. The config has several fields, including one that's called `endpoint_url` which just makes sense. Is there something particular you want me to change?

Reply:
I... Think I may have looked at an old change. Looks good!